src/app/lib/config/services/bengali.js

```diff
@@ -30,7 +30,7 @@ const fonts = isTest()
     ]
   : [F_SHONAR_BANGLA_BOLD, F_SHONAR_BANGLA_REGULAR];
 
-const script = isTest() ? bengali : bengaliNotoSerif;
+const script = isTest() ? bengaliNotoSerif : bengali;
```

Comment:
I should have noticed this in code review! Too early in the morning and too distracted by the weird font function array thing 😆